front-end/src/Employee/Registration/employee-registration.component.tsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import { Formik, Form, FormikHelpers } from 'formik';
+import dayjs from 'dayjs';
+import {
+  Button,
+  TextInput,
+  DatePicker,
+  DatePickerInput,
+  RadioButton,
+  RadioButtonGroup,
+  Column,
+} from 'carbon-components-react';
+import './employee-registration.scss';
```

Comment:
Better to use css modules in my opinion its better since it doesn't affect the globally applied css have a look at this [article](https://medium.com/@ralph1786/using-css-modules-in-react-app-c2079eadbb87)

`import styles from './employee-registration.scss'`